index.html

```diff
@@ -13,11 +13,30 @@
     <link rel="stylesheet" href="styles/main.css">
     <link rel="stylesheet" href="styles/footer.css">
   </head>
-  <body>
-    <div class="div--maincolor">
-      <h1 class="h1">My First Webpage</h1>
-      <p class="p">Hello World!</p>
-      <img src="img/header/logo_prtimes.svg" alt="PR TIMES"/>
-    </div>
+  <body class="container">
+    <header class="container__header header">
+      <div class="header-content header__header-content">
+        <div class="title header-content__title">
+          <img class="title__logo" src="img/header/logo_prtimes.svg" alt="PR TIMES"/>
+          <h1 class="title__text">プレスリリース・ニュースリリース配信サービスのPR TIMES</h1>
+        </div>
+        <ul class="header-content__menu menu--bgblue">
+          <li class="menu__item"><a href="#">プレスリリースを受信</a> |</li>
+          <li class="menu__item"><a href="#">企業登録申請</a> |</li>
+          <li class="menu__item"><a href="#">管理画面</a></li>
+        </ul>
+        <form class="keyword header-content__search">
+          <input class="keyword__input" type="text" placeholder="キーワードで検索" aria-label="キーワードで検索">
+          <button class="keyword__button" type="submit"></button>
+        </form>
```

Comment:
こちらは、`header-content__search`は`header`の`search`のものの名前、もしくは、`header-content__keyword`。formの再利用できるスタイル用の名前は`keyword`はなく、`form`した方が良いと思います。
```suggestion
        <form class="form header-content__search">
          <input class="form__input" type="text" placeholder="キーワードで検索" aria-label="キーワードで検索">
          <button class="form__button" type="submit"></button>
        </form>
```

Reply:
[こちらのcommit](https://github.com/ShunDeveloper/prtimes_htmlcss/pull/10/commits/9ef67ddf18c10ba693b35a666b2c996c16f36fc7)で修正しました. `keyword`を`form`に変更しました. また, 再利用している`form__input`, `form__button`についても変更を加えました.